index.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+<link rel="stylesheet" href="styles.css">
+</head>
+<body>
+<div id="main">
+<div id="title-div">
+    <h1 id="title">Dr Norman Borlaug</h1>
+    <p id="saver"> The man who saved a billion lives</p>
+</div>
+<div id="img-div">
+    <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+    <div id="tribute-link"></div>
+    <div id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</div>
+</div>
+    <h3 id="tribute-info">Here's a time line of Dr. Borlaug's life:</h3>
+    <div id="timeline-div" style="text-align: center;">
+        
+    </div>
+    <h4 id="tribute-link" href="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" target ="_blank">
+        If you have time, you should read more about this incredible human being on his Wikipedia entry.</h4>
+<div>
+<ul id="dots">
```

Comment:
I think you should use "class" instead of "id" because a class is used to style multiple items and elements, but with id, you can change the CSS property of only one element.

Reply:
- [ ] Very nice usage of inline comments. please keep it up while doing reviews for your partners and colleagues.